### ISSUE TITLE
Loss function as config

### ISSUE THREAD

**Issue by lvoegtlin** [DIVA-DIA/DIVA-DAF#14]
**Is your feature request related to a problem? Please describe.**
Make it possible to define the loss function in the config. 

**Describe the solution you'd like**
Define some defaults functions and create a config for them. Then hand over the criterion object to the task at the beginning of the training.

### Checklist

- [ ] define 4 basic losses (Xentropy, L1, MSE, BCE)
- [ ] create configs
- [ ] hand over the loss function as a parameter to the task
